### ISSUE TITLE
Quart fails to launch

### ISSUE THREAD

**Issue by mitchbte** [Nuitka/Nuitka#331]
I'm trying to use Nuitka with Quart. The compilation works fine but the app fails to launch. Here is a small sample to test.

Note Quart sits on top of Flask, I believe this issue will exist with Flask as well.

Versioning
```
(venv) PS E:\projects> nuitka --version
0.6.2
Python: 3.7.2 (tags/v3.7.2:9a3ffc0492, Dec 23 2018, 23:09:28) [MSC v.1916 64 bit (AMD64)]
Executable: E:\projects\venv\Scripts\python.exe
OS: Windows
Arch: x86_64
```

Required packages
```
pip install quart
```

Compile command used
```
python -m nuitka --show-progress --warn-unusual-code --show-modules --show-scons .\we
b_test.py
```

Sample Code
```
from quart import Quart, websocket

app = Quart(__name__)

@app.route('/')
async def hello():
    return 'hello'

@app.websocket('/ws')
async def ws():
    while True:
        await websocket.send('hello')

app.run()

```

Thanks for a great tool.
Cheers. Mitch

**Comment by kayhayen** [comment]
I tried this on Linux first, and there it just worked fine. Will try on Windows later.

**Comment by kayhayen** [comment]
I think this might be causes by attempts to scan what is actually compiled code. Windows hates attempt to   look up file named `<frozen>`. Not sure if the coroutine errors after that are caused by that, but I would think so.

Unsure what to think of that, might need an upstream change to handle this, but maybe Nuitka is at fault, I will try and check what happens by making the frozen name include more information.

**Comment by mitchbte** [comment]
I think that's correct. I feel like it should be possible (provided it isn't optimized away). I haven't looked into how nuitka exposes module data so I'm not sure how you could approach it.

Did you have any luck by adding the name.

**Comment by kayhayen** [comment]
Well, actually I noticed, that Nuitka is doing it far and few, and only in 2 cases, one which ought to not happen ever. And one time for a pseudo module that hides a `types` patch, where `types._GeneratorWrapper` can get massaged. And low and behold, if I change that from `<frozen>` to `frozen` it just works.

So, seems somebody in Python land near Quart insists on walking all the `sys.modules` and using it's `__file__` and expects it to be usable.

The way to rectify this, is of course to remove the pseudo module after it did its job from that visibility, where I probably shouldn't be anyway.

**Comment by kayhayen** [comment]
So this should be solved now, going to be in the next pre-release, check it out under http://nuitka.net/doc/factory.html

**Comment by kayhayen** [comment]
It's now on develop, and will be part of 0.6.4 eventually.

**Comment by kayhayen** [comment]
Released as 0.6.4
